crates/vm/levm/src/precompiles.rs

```diff
@@ -173,11 +174,27 @@ fn identity(
 }
 
 fn sha2_256(
-    _calldata: &Bytes,
-    _gas_for_call: U256,
-    _consumed_gas: &mut U256,
+    calldata: &Bytes,
+    gas_for_call: U256,
+    consumed_gas: &mut U256,
 ) -> Result<Bytes, VMError> {
-    Ok(Bytes::new())
+    let data_size: u64 = calldata
```

Comment:
Why not use a `usize` instead?

Reply:
You'll end up doing the conversion anyways but doing it inside of the gas cost function its cleaner in my opinion